index.html

```diff
@@ -20,7 +20,30 @@ <h1>Hotel reservation</h1>
 
     <div class="site-content">   
         <!-- Site Content -->
-
+		
+		
+	<div class="search-form-content custom-form">
+		<div class="background">
+			<div class="shape"></div>
+			<div class="shape"></div>
+		</div>
+		<form>
+			<h3>Search Here</h3>
+	
+			<label for="check-in">Check-in:</label>
+			<input type="date" id="check-in" name="check-in">
+	
+			<label for="check-out">Check-out:</label>
+			<input type="date" id="check-out" name="check-out">
+	
+			<label for="people">Numbers of people:</label>
+			<input type="number" min="1" id="people" name="phone-number">
```

Comment:
Are you sure that is the right name ? (phone-number) ?

Reply:
Right,  fixed (name="people").